File: Proyecto Escolar - Pagina Lenguajes de Programacion/tareas/ruby/Tarea_8.rb
=begin
 * Alumno: Brandon Jahir Rojas Quintanilla
 * Matricula: 1635448
 * Hora: N2
 * Salón: 4201
 * Función: Convierte de Celsius a Fahrenheit y viceversa
=end

puts("¿Que conversion quieres realizar?") 
puts("0.- Celsius a Fahrenheit\n1.- Fahrenheit a Celsius")
print("\nSeleccione una opcion: ")
eleccion = gets().to_i 

case eleccion 
	when 0
		print("Ingrese los grados Celsius: ")
		celsius = gets().to_i
		fahrenheit =(celsius * (9/5)) + 32;
		print(celsius.to_s + "°C equivalen a " + fahrenheit.to_s + "°F")# .to_s convierte a String#
	when 1
		print("Ingrese los grados Fahrenheit: ")
		fahrenheit = gets().to_i
		celsius = (fahrenheit - 32) * (5/9);
		print(fahrenheit.to_s + "°F equivalen a " + celsius.to_s + "°C")		
	else
		print("Opción invalida")
end 
	

File: Proyecto Escolar - Pagina Lenguajes de Programacion/tareas/java/Tarea_8.java
/* 
 * Alumno: Brandon Jahir Rojas Quintanilla
 * Matricula: 1635448
 * Hora: N2
 * Salón: 4201
 * Función: convierte de grados centigrados a grados fahrenheit y viceversa
 */ 
 
import javax.swing.JOptionPane;
public class Tarea_8{
	public static void main(String[] args) {
		
		String entrada;
		int celsius, eleccion, fahrenheit;
		
		eleccion = JOptionPane.showOptionDialog(null, "Que conversion quieres realizar?", "Temperatura", JOptionPane.DEFAULT_OPTION, 
			JOptionPane.QUESTION_MESSAGE, null, new String[]{"°C a °F", "°F a °C"}, null);
		
		switch(eleccion) {
			case 0: 
				entrada = JOptionPane.showInputDialog("Ingrese los grados Celsius");
				celsius = Integer.parseInt(entrada);
				fahrenheit = (celsius * (9/5)) + 32; 
				JOptionPane.showMessageDialog(null,celsius + "°C equivalen a " + fahrenheit + "°F");
				break;
			case 1: 
				entrada = JOptionPane.showInputDialog("Ingrese los grados Fahrenheit");
				fahrenheit = Integer.parseInt(entrada);
				celsius = (fahrenheit - 32) * (5/9);
				JOptionPane.showMessageDialog(null,fahrenheit +"°F equivalen a "  + celsius + "°C");
				break;
			default: JOptionPane.showMessageDialog(null,"Opcion Invalida");
			break;
		}
			
	} 
} 